src/commands/DensityProfile.cpp

```diff
@@ -34,7 +36,7 @@ For more information about chemfiles selection language, please see
 Options:
   -h --help                     show this help
   -o <file>, --output=<file>    write result to <file>. This default to the
-                                trajectory file name with the `_dp.dat` extension.
+                                trajectory file name with the `.density.dat` extension.
```

Comment:
Could you move `extension` to the next line? It looks like it goes after the 80 chars limit.